scripts/fetch-release-changelog.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as fs from 'fs';
+import * as process from 'process';
+import * as https from 'https';
+
+/**
+ * Take the `pullRequestNumber` output of the `changesets/action` GitHub action
+ * on standard input and return the CHANGELOG from the PR description in stdout.
+ *
+ * https://github.com/changesets/action#outputs
+ */
+
+const pullRequestNumber = fs.readFileSync(process.stdin.fd, 'utf-8')?.trim();
```

Comment:
Could this be provided as `env` in the run step and read from `process.env`?
Might also want to throw if this is undefined.

Reply:
+1 added verification and throwing. Since we don't need the pr number anywhere else, I've opted to keep as a passed in value.